client/src/components/Meeting/Meeting.tsx

```diff
@@ -1,66 +1,112 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
 
 // Material-UI and Style
 import Typography from '@material-ui/core/Typography';
-import { Button, Grid, GridSpacing, Paper } from '@material-ui/core';
+import { Box, Button, Grid, Paper } from '@material-ui/core';
 import useStyles from '../Meeting/useStyles';
 import SettingsIcon from '@material-ui/icons/Settings';
 import Divider from '@material-ui/core/Divider';
 import ScheduleIcon from '@material-ui/icons/Schedule';
 import AddIcon from '@material-ui/icons/Add';
+import fetchEvents from './../../helpers/APICalls/fetchEvents';
 
 // Components
 import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
 
 // Interface
 import { User } from '../../interface/User';
+import { useSnackBar } from '../../context/useSnackbarContext';
 
 interface Props {
   loggedInUser: User;
   handleDrawerToggle?: () => void;
 }
 
+const circularColorPicker = (index: number) => {
+  const colors = ['#7900FF', '#89B800', '#FF7000'];
+  const currentIndex = index % 3;
+  return colors[currentIndex];
+};
+
 const Meetings = ({ loggedInUser }: Props): JSX.Element => {
+  const { updateSnackBarMessage } = useSnackBar();
   const classes = useStyles();
-  const [spacing] = React.useState<GridSpacing>(2);
+  const [events, setEvents] = useState([]);
+
+  useEffect(() => {
+    fetchEvents().then((data) => {
+      if (data.success) {
+        setEvents(data.success);
+      } else {
+        updateSnackBarMessage('Could not recieve events from backend server');
+      }
+    });
+    // eslint-disable-next-line
+  }, []);
 
+  const copyToClipBoard = (url: string) => {
+    navigator.clipboard.writeText(url);
+  };
   return (
-    <Grid container spacing={3}>
-      <Grid item xs={12}>
-        <Divider />
-        <AvatarDisplay loggedIn user={loggedInUser} />
-        <Typography variant="h5">{loggedInUser.username}</Typography>
-        <Typography variant="h5">{loggedInUser.email}</Typography>
-        <Button variant="outlined" color="primary" href="#outlined-buttons" className={classes.right}>
-          <Typography variant="h6">
+    <Box>
+      <Grid container xs={12} direction="row" className={classes.profileSection}>
+        <Box display="flex">
+          <Box>
+            <AvatarDisplay loggedIn user={loggedInUser} />
+          </Box>
+          <Box marginLeft={2}>
+            <Typography component="h3" className={classes.title}>
+              {loggedInUser.username}
+            </Typography>
+            <Typography variant="subtitle1">{loggedInUser.email}</Typography>
+          </Box>
+        </Box>
+        <Box>
+          <Button variant="outlined" color="primary" className={classes.newEventButton}>
             <AddIcon />
-            New Event
-          </Typography>
-        </Button>
+            <Typography variant="h6">New Event</Typography>
+          </Button>
```

Comment:
Button API has a `startIcon` property, you should change this to utilize the library more.